client/src/modules/creditors/groups/creditor-groups-list.html

```diff
@@ -1,12 +1,39 @@
 <div class="container">
-  <div class="row">
+  <div class="row" ng-show="CreditorGroupCtrl.creditorGroupList.length>0">
     <div class="col-md-8 col-md-offset-2">
 
-      <div ng-repeat="cg in CreditorGroupCtrl.creditorGroupList | orderBy: 'name' track by $index">
+      <div class="panel panel-default" style="margin-bottom : 0px;">
+        <div class="panel-heading clearfix" style="text-align : left">
```

Comment:
Instead of using `text-align: left`, it is better to use classes that we can change using `less` later.  There is a class for this: [text-left](https://getbootstrap.com/docs/3.3/css/#type-alignment).